packages/xstate-react/test/useActor.test.tsx

```diff
@@ -48,6 +56,7 @@ describeEachReactMode('useActor (%s)', ({ suiteKey, render }) => {
                 return event.output;
               }
             }),
+            // TODO: this should be typed as { output: string }
```

Comment:
If you'd remove explicit generics from this `createMachine` call then this would be typed correctly. This happens because your actor-related types on `config.types` are just ignored (since you provided explicit generics and didn't pass actor-related type there)